dowhy/causal_estimator.py

```diff
@@ -679,6 +685,91 @@ def signif_results_tostr(self, signif_results):
         return s
 
 
+def estimate_effect(
+    treatment: Union[str, List[str]],
+    outcome: Union[str, List[str]],
```

Comment:
data should be one of the arguments for estimate_effect. Actually it should be the first argument. Just like sklearn, the `estimate_effect` method of an Estimator object (or later, the fit method) should be passed the data.

Reply:
I'll add this in a future PR when I start to refactor all the estimator objects